operate/qa/integration-tests/src/test/java/io/camunda/operate/elasticsearch/FinishedImportingIT.java

```diff
@@ -333,6 +333,28 @@ final var record = generateRecord(ValueType.PROCESS_INSTANCE, "8.6.0", 1);
     assertImportPositionMatchesRecord(record, ImportValueType.PROCESS_INSTANCE, 1);
   }
 
+  @Test
+  public void shouldWriteDefaultEmptyDefaultImportPositionDocumentsOnRecordReaderStart() {
+    recordsReaderHolder.getAllRecordsReaders().stream()
+        .map(ElasticsearchRecordsReader.class::cast)
+        .forEach(ElasticsearchRecordsReader::postConstruct);
```

Comment:
❌ should not be needed. if the beans are marked as components, spring will call the `@PostConstruct` annotated methods after initializing the bean.

If something is not working correctly, this needs to be investigated rather than worked around. there's a possibility that it won't work in production if it's not working in (a spring managed) IT.

Reply:
The beans are only instantiated once for the test (which is what we expect the default behaviour to be) changing this would require deeper changes to how all operate ITs are run which we take a lot longer.

Reply:
if you try it yourself, put a breakpoint in the post construct function and debug the entire `FinishedImportingIT` file you will see for each record reader is only constructed once

Reply:
would annotating the test method you need a clean context for with `@DirtiesContext(methodMode = MethodMode.BEFORE_METHOD)` help?

Reply:
It also resets the operate properties and causes cascading issues because other components also rely on the properties to access the indices (mainly the prefix values). Will just move on for now